### ISSUE TITLE
VisVis with PyQt5

### ISSUE THREAD

**Issue by JarrettR** [UmSenhorQualquer/pyforms#53]
This doesn't run.

There are three causes: 

1. ControlVisVis loading the default conf and then immediately checking it (default is set to PyQt4)
https://github.com/UmSenhorQualquer/pyforms/blob/master/pyforms/gui/Controls/ControlVisVis.py#L22
 I don't know offhand how to elegantly solve this.

2. Base object QtGUI being referenced instead of the child
https://github.com/UmSenhorQualquer/pyforms/blob/master/pyforms/gui/Controls/ControlVisVis.py#L31
Solved by simply removing the `QtGui.` in front of `QtGui.QWidget()` and other instances

3. Passing VisVis hardcoded Qt4 string
https://github.com/UmSenhorQualquer/pyforms/blob/master/pyforms/gui/Controls/ControlVisVis.py#L40
Also easily solved by adding the `if conf.PYFORMS_USE_QT5:` conditional to `self._app = vv.use('pyqt4')`

**Comment by abr123** [comment]
I totally agree with @JarrettR . No.2 and 3 changes are small, but can help a lot.
